mmtk/src/collection.rs

```diff
@@ -100,6 +104,17 @@ impl Collection<JuliaVM> for VMCollection {
     fn vm_live_bytes() -> usize {
         crate::api::JULIA_MALLOC_BYTES.load(Ordering::SeqCst)
     }
+
+    fn is_collection_enabled() -> bool {
+        unsafe {
+            AtomicU32::load(
```

Comment:
As you declare the variable as `AtomicU32`, can you simply access it rather than getting the address and transmute it?